3570/3/rating_places/app/models/comment.rb

```diff
@@ -0,0 +1,13 @@
+class Comment < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :place, counter_cache: true
+
+  after_create :update_place_rating
+
+  private
+
+  def update_place_rating
```

Comment:
Why are you updating rating of place in comment model?

Reply:
rating update occurs after creating a new comment

Reply:
So? You making changes on Place on Comment model?

Reply:
denormalized data(average rating) depends on child model.

Reply:
Again, this method is update PLACE. why this method not in PLACE model? If you need it, you can delegate it to comments (you dont even need this, as comment_count and rating columns)